### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by kaysabelle**
Added nice things.
Deleted not-nice things.
Moved things around.

### CODE REVIEW

File: radlibDemoApp/config.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <splash src="res/screen/android/splash-port-mdpi.png" density="port-mdpi"/>
     <splash src="res/screen/android/splash-port-xhdpi.png" density="port-xhdpi"/>
 </platform>
-<widget id="com.example.lfdemo" version="0.0.1" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
```
Comment (MinickLS) [inline radlibDemoApp/config.xml:13]: nice catch

File: radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/RadLib.js [hunk 2]
```diff
@@ -1,8 +1,3 @@
-//These comments are extremely obviously who writes in javascript/nodejs, so
```
Comment (MinickLS) [inline radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/RadLib.js:1]: Better catch

File: radlibDemoApp/www/js/index.js [hunk 3]
```diff
@@ -220,16 +248,62 @@ function finishdelREADER() {
...
-      bluetoothUtils.turnOnBluetooth(dumpLog, dumpLog);
-   } else {
-      bluetoothUtils.turnOffBluetooth(dumpLog, dumpLog);
-   }
+
+
+
+
+
+/* TESTING FUNCTIONS FOR DEBUGGING *?
```
Comment (jeslin) [inline radlibDemoApp/www/js/index.js]: lol change the ? to /

Comment (kaysabelle) [inline radlibDemoApp/www/js/index.js]: Catch of the day :fish: 

Fixed.

**Comment by jeslin** [comment]
:+1:

**Pull request merged**